content/docs/use-cases/versioning-data-and-model-files/index.md

```diff
@@ -26,82 +27,62 @@ Here are some benefits of this approach:
   repository history. Security audits are also made possible by an immutable
   history of changes in data and models.
 - Low coupling: separate code from data by caching large files/directories
-  automatically. This makes your project easier to maintain (high cohesion) and
+  automatically. This makes data projects easier to maintain (high cohesion) and
   improves data persistence.
 - Simple interface: work with simple terminal [commands](/doc/command-reference)
   similar to `git`.
-- Treating data as code also allows for other advanced features, see
+- Treating _data as code_ also allows for other advanced features, see
   [Get Started](/doc/start) for a primer.
 
-## How it looks
-
-DS/ML are iterative
-
-Replace large data files and directories with tiny, human-readable metafiles
-that can be versioned with Git.
-
-A
-[Content-addressable storage](https://www.google.com/url?q=https://en.wikipedia.org/wiki/Content-addressable_storage&sa=D&ust=1603526252385000&usg=AOvVaw3Y4fV6jAM2grfE4k9AP3HX)
-(<abbr>DVC Cache</abbr>) is created automatically.
-
-Metafiles link data from cache to workspace.
-
-Use Git commands to create versions (commits), branches, etc. git+dvc checkout
-to switch between versions
-
-Versioning large data files and directories with Git is made possible by
-replacing them with small, human-readable
-[metafiles](/doc/user-guide/dvc-files-and-directories). The data itself is
-<abbr>cached</abbr> outside of Git (locally or externally), and can be
-synchronized automatically with
-[dedicated storage](/doc/use-cases/versioned-storage) — a good way to share data
-for collaboration.
+## How it looks & feels
```

Comment:
BTW is this a good title? "look & feel" is kind of a design term, I think.

Reply:
Actually, I think it's good: "looks" refers to the worspace files, dvc cache, etc, while "feels" refers to the DVC/Git workflow. Resolving